presto-raptor/src/main/java/com/facebook/presto/raptor/filesystem/RaptorHiveHdfsConfiguration.java

```diff
@@ -47,6 +49,11 @@
 public class RaptorHiveHdfsConfiguration
         implements RaptorHdfsConfiguration
 {
+    static {
```

Comment:
Just curious: what are these changes doing?

Reply:
It is a necessary step for HDFS to get the right file system. Check `HdfsEnvironment`